src/main/java/biblioteka/Calendar.java

```diff
@@ -0,0 +1,354 @@
+package biblioteka;
+import model.Thing;
+import model.User;
+import java.sql.*;
+import java.util.List;
+//https://www.postgresqltutorial.com/postgresql-jdbc/ - fajny tutorial, sa tam tez metody do update'u czy kasowania ale narazie niepotrzebne
+//https://www.postgresqltutorial.com/postgresql-jdbc/query/ - na podstawie tego wyszukiwanie Userow zrobione
+//https://www.postgresqltutorial.com/postgresql-jdbc/insert/ - na podstawie tego dodawanie Userow zrobione
+
+public class Calendar {
+
+    private static String URL = "jdbc:postgresql://localhost:8012/event_base";
+    private static String USER = "postgres";
+    private static String PASSWORD = "password";
+
+    private Connection conn;
+    private Statement stat;
+
+    public Calendar() {
+        try {
+            conn = DriverManager.getConnection(URL, USER, PASSWORD);
+            stat = conn.createStatement();
+            System.out.println("Connected with database");
+        } catch (SQLException e) {
+            System.err.println("Problem during connection");
+            e.printStackTrace();
+        }
+        createTables();
```

Comment:
I think it should be wrapped in try like getting connection. There is a rule:

> throw early, catch late

Reply:
Structure needs discussion probably.

Reply:
> I think it should be wrapped in try like getting connection. There is a rule:
> 
> > throw early, catch late

It is done later in body of the createTables() method. It looks like this:

 public boolean createTables()
    {
        String createUsers = "CREATE TABLE IF NOT EXISTS users(user_id serial primary key, name varchar, surname varchar)";
        String createThings = "CREATE TABLE IF NOT EXISTS things(thing_id serial primary key, name varchar)";

        try {
            sqlStatement.execute(createUsers);
            sqlStatement.execute(createThings);
        } catch (SQLException e) {
            System.err.println("Error during creating table");
            e.printStackTrace();
            return false;
        }
        return true;
    }

Reply:
I deleted old method createTables(), now I'm doing everything in Calendar contructor., using batch processing. Should it much better from now.